### DISCUSSION TITLE
PS Vita version

### DISCUSSION THREAD

**Discussion by landsman** [localsend/localsend#1564] [Discussion (English, International)] [answered]
It would be lovely to have this app on Vita. 
Moving things via USB cable is just annoying. 
Community is really rocks right now: 
- https://www.reddit.com/r/vita/
- https://www.reddit.com/r/VitaPiracy/

I love this app it on my macOS <> linux <> android devices.
Thanks for your work!

**Comment by Tienisto** [comment, marked as answer]
[Flutter](https://flutter.dev/) doesn't support PS Vita so it's very difficult to make it happen.

**Comment by resashkin** [comment]
Hey guys! Last august I'd accepted challenge to migrate Flutter to PS Vita. More details here about my first attempt with Flutter Example project - https://www.reddit.com/r/vitahacks/comments/1vlswp1/flutter_ps_vita_poc_is_alive/

After that I decided to migrate localsend and fix almost every part API in Flutter - still in progress...

But core part with sending/receiving is working already (with support TLS)

https://github.com/user-attachments/assets/577d2bbd-9c60-42a3-9cae-1c17700b7884

**Reply by resashkin** [reply]
cc: @Tienisto @landsman
